src/com_tjvendors/admin/views/vendorfee/tmpl/edit_bs2.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * @package     TJVendors
+ * @subpackage  com_tjvendors
+ *
+ * @author      Techjoomla <extensions@techjoomla.com>
+ * @copyright   Copyright (C) 2009 - 2019 Techjoomla. All rights reserved.
+ * @license     http://www.gnu.org/licenses/gpl-2.0.html GNU/GPL
+ */
+
+defined('_JEXEC') or die();
+
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Router\Route;
+
+HTMLHelper::addIncludePath(JPATH_COMPONENT . '/helpers/html');
+HTMLHelper::_('bootstrap.tooltip');
+HTMLHelper::_('behavior.formvalidator');
+HTMLHelper::_('formbehavior.chosen', 'select');
```

Comment:
@Sachintechjoomla Is chosen needed?

Reply:
Yes we need this in joomla 3 for display chosen than select

Reply:
> Yes we need this in joomla 3 for display chosen than select